api/http/handler/edgegroups/edgegroup_list.go

```diff
@@ -92,9 +93,15 @@ func getEdgeGroupList(tx dataservices.DataStoreTx) ([]decoratedEdgeGroup, error)
 			return nil, httperror.InternalServerError("Unable to retrieve environment types for Edge group", err)
 		}
 
+		trustedEndpointsCount, err := getTrustedEndpointsCount(tx, edgeGroup.Endpoints)
```

Comment:
I think we should filter `edgeGroup.Endpoints` to show only trusted endpoints. this will make sure this bug won't happen in another place.

btw, I was going to link to edgegroup_inspect to show how it was done there, but it seems like filtering only happens for dynamic groups inside `GetEndpointsByTag`. this should be fixed there too